html/pfappserver/root/src/views/Configuration/radius/general/_components/TheForm.vue

```diff
@@ -62,6 +62,11 @@
       :text="$i18n.t('Enables a Redis driven cache for NTLM authentication.In order for this to work, you need to setup proper NT hash syncronization between your PacketFence server and your AD. Refer to the Administration guide for more details. Applying this requires a restart of radiusd.')"
     />
 
+    <form-group-process-bandwidth-accounting namespace="process_bandwidth_accounting"
+      :column-label="$i18n.t('Process Bandwidth Accounting')"
+      :text="$i18n.t('Enables processing of bandwidth accounting.')"
```

Comment:
This seems to require a restart of pfacct to be effective. If that's the case, add it to this label and add the restart button of pfacct after the pages finishes saving